06.bowling_object/shot.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class Shot
+  def initialize(score)
+    @score = score
+  end
+
+  def point
+    @score == 'X' ? 10 : @score.to_i
```

Comment:
```suggestion
    strike? ? 10 : @score.to_i
```

個人的にはこうした方が可読性が高いかなーと思います